janitor/functions/pivot.py

```diff
@@ -935,8 +923,10 @@ def _pivot_longer_dot_value(
     """
     if np.count_nonzero(mapping.columns == ".value") > 1:
         outcome = mapping.pop(".value")
-        outcome = outcome.agg("".join, axis=1)
-        mapping[".value"] = outcome
+        out = outcome.iloc[:, 0]
+        for _, val in outcome.iloc[:, 1:].items():
+            out += "" + val
```

Comment:
I'm a little lost, why is the `""` necessary here?  
Does this cover some edge case that the previous code doesn't cover?

Otherwise, I would think the intention of the old code is clearer and we can avoid iteration with the for-loop.

Reply:
Speed. Wayy faster. Discovered this by accident when answering a question on SO. The speed difference between the for loop and agg especially for significantly large arrays is significant

Reply:
interesting, thanks. I'll have to look into that.

And the `""`? what's that for?

Reply:
ahh my bad ... I put the "" as a separator; totally unnecessary. thanks for that